unity/Assets/Scripts/AgentManager.cs

```diff
@@ -319,11 +322,6 @@ public BaseFPSAgentController ReturnPrimaryAgent()
 		ResetSceneBounds();
 	}
 
-	// If fov is <= min or > max, return defaultVal, else return fov
-	private float ClampFieldOfView(float fov, float defaultVal = 90f, float min = 0f, float max = 180f) {
-		return (fov <= min || fov > max) ? defaultVal : fov;
-	}
-
```

Comment:
Now, instead of clamping the field of view, we provide an error if the field of view is out of bounds.